ext/native/thin3d/VulkanRenderManager.cpp

```diff
@@ -616,8 +616,27 @@ void VulkanRenderManager::Finish() {
 
 void VulkanRenderManager::Wipe() {
 	int curFrame = vulkan_->GetCurFrame();
-	for (auto iter : steps_) {
-		delete iter;
+	for (auto step : steps_) {
+		// Need to release held framebuffers.
+		switch (step->stepType) {
+		case VKRStepType::RENDER:
```

Comment:
Maybe this belongs better in queue runner... not sure.

-[Unknown]

Reply:
Eh, it's fine here. We can move it later if we feel like it.